apps/tangle-dapp/app/liquid-staking/[tokenSymbol]/page.tsx

```diff
@@ -24,86 +18,41 @@ const LiquidStakingTokenPage: FC<Props> = ({ params: { tokenSymbol } }) => {
     value.toString(),
   );
 
-  // Invalid token provided on the URL parameters.
+  // Invalid token on the URL.
   if (!possibleTokens.includes(tokenSymbol)) {
-    return notFound();
+    return redirect(PagePath.LIQUID_STAKING);
```

Comment:
Seems like the outdated version was chosen during merge conflict resolution by mistake